src/muc_light/mod_muc_light_db_mnesia.erl

```diff
@@ -131,6 +132,10 @@ remove_user(UserUS, Version) ->
     {atomic, Res} = mnesia:transaction(fun remove_user_transaction/2, [UserUS, Version]),
     Res.
 
+-spec remove_domain(mongooseim:host_type(), jid:lserver(), jid:lserver()) -> ok.
+remove_domain(_HostType, _RoomS, _LServer) ->
+    ok.
```

Comment:
Just a note: we might mark Mnesia support as TODO somewhere not to forget about it

Reply:
I will not forget :D